demos/browser/app/meeting/meeting.ts

```diff
@@ -129,7 +130,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
 
   // feature flags
   enableWebAudio = false;
-
+  logger: any = null;
```

Comment:
- Do not need an instance variable since `this.logger` is being used in one block `{}`.

- If you want this variable, change the type `any` => `ConsoleLogger | MeetingSessionPOSTLogger`.